build-conf/zCEE3.properties

```diff
@@ -0,0 +1,20 @@
+# Releng properties used by language/zCEE3.groovy
+
+#
+# Comma separated list of required build properties for zCEE3.groovy
```

Comment:
You are missing to add the properties file to the `buildPropFiles` list in `build-conf/build.properties`

Reply:
Added now